setup.py

```diff
@@ -37,6 +37,10 @@ def get_version(rel_path):
 
 HERE = pathlib.Path(__file__).parent
 README = (HERE / "README.md").read_text()
+if os.path.exists("keras_nlp/version.py"):
```

Comment:
Why the if? Can't we just use the new path here?

Reply:
We have it in the Keras implementation but we probably don't need since moving forward we should be checking `version.py`. Removing it!